### ISSUE TITLE
Method requiest: Multifractal (IAAWT) surrogates

### ISSUE THREAD

**Issue by Datseris** [JuliaDynamics/TimeseriesSurrogates.jl#71]
Section 5.2 of the Lancaster review:

> The discrete wavelet transform has also been used as the basis of multifractal surrogates, which aim to enable testing of nonlinear interdependence within, with, between or among time series obtained from multifractal processes [113]. Multifractal surrogates preserve the multifractal properties of input data, i.e. interactions among scales and nonlinear dependence structures [113]. This approach was developed further by Keylock [114] and named iterated amplitude adjusted wavelet transform (IAAWT) surrogates. IAAWT surrogates are designed to preserve both, the multifractal characteristics of the time series and also the distribution of values in the time series using the iterative approach used in IAAFT surrogates. Multifractal surrogates have been used to test cross-scale interactions in atmospheric dynamics [115].
5.3.

**Comment by kahaaga** [comment]
This is the same surrogate method as in #64. Closing in favor of that.

**Comment by kahaaga** [comment]
> This is the same surrogate method as in #64. Closing in favor of that.

This is not true. The IAAWT surrogate method is an extension developed by Keylock to the random cascade method by Palus in (#64)
